common/app/staticpages/StaticPages.scala

```diff
@@ -19,7 +19,7 @@ object StaticPages {
       section = Option(SectionSummary(id="email-signup-page", activeBrandings=None)),
       webTitle = webTitle,
       analyticsName = "email-signup-page",
-      contentType = "signup",
+      contentType = "Signup",
```

Comment:
capitalisation appears to be the more common approach in other pages: 
`contentType = "Interactive"` so I was mostly going with the convention of the masses 😅 

in meta.scala the approach is to use `contentType.toLowerCase` when doing string comparisons, probably to avoid the CaSe issUes ¯\\\_(ツ)_/¯